fixed-price-subscriptions/client/vanillajs/account.js

```diff
@@ -19,7 +19,7 @@ document.addEventListener('DOMContentLoaded', async () => {
       </p>
 
       <p>
-        Card last4: ${subscription.default_payment_method.card.last4}
+        Card last4: ${subscription.default_payment_method?.card?.last4}
```

Comment:
Ran into a race condition here. Retrieving the subscription too quickly can return it before our webhook consumer has set the default payment method. I think it's worth the tradeoff.

Reply:
ack